xarray/core/duck_array_ops.py

```diff
@@ -101,6 +101,8 @@ def isnull(data):
         return isnan(data)
     elif issubclass(scalar_type, (np.bool_, np.integer, np.character, np.void)):
         # these types cannot represent missing values
+        if hasattr(data, "_meta"):
```

Comment:
We have `is_duck_dask_array` somewhere. But I'm not sure if `pint` (for e.g.) forwards the `_meta` property. cc @keewis

Reply:
`pint` will forward to the wrapped array using `__getattr__` if it doesn't define the attribute itself, so we should be good. I'm not sure if other duck dask arrays do that, though (are there any rules/conventions for this?).